### ISSUE TITLE
TA lacking eventgen

### ISSUE THREAD

**Issue by randolpht** [PaloAltoNetworks/Splunk_TA_paloalto#16]
Are you guys not including eventgen in this revision of the TA?

**Comment by btorresgil** [comment]
We removed the eventgen because its so big.  At 30 MB, it seemed silly to redistribute that across ever SearchHead, Indexer, and Heavy Forwarder.  The eventgen comes packaged with the docker version of the app:  https://hub.docker.com/r/btorresgil/splunk-panw-demo

Do you want it added back into the TA?  What's the use case?

**Comment by randolpht** [comment]
Nope. I just have done some stuff for my internal lab with the eventgen. I will grab it from your docker version.

Thanks!
